protocol/src/main/java/org/apache/drill/exec/proto/beans/CoreOperatorType.java

```diff
@@ -58,7 +58,8 @@
     HBASE_SUB_SCAN(33),
     WINDOW(34),
     NESTED_LOOP_JOIN(35),
-    AVRO_SUB_SCAN(36);
+    AVRO_SUB_SCAN(36),
```

Comment:
I agree that the situation is less than ideal, and it has always kind of bothered me. But I think this is a separate issue and really has no bearing on the current PR. We should move this discussion to the mailing list or a new JIRA.